torchvision/ops/feature_pyramid_network.py

```diff
@@ -64,6 +82,12 @@ def __init__(self, in_channels_list, out_channels, extra_blocks=None):
         if extra_blocks is not None:
             assert isinstance(extra_blocks, ExtraFPNBlock)
         self.extra_blocks = extra_blocks
+        self.last_inner_block = self.inner_blocks[-1]
```

Comment:
This still breaks loading of old state dicts, together with
```python
self.inner_blocks = self.inner_blocks[:-1][::-1]
self.layer_blocks = self.layer_blocks[:-1][::-1]
```
because the state_dict is loaded after model instantiation.

Reply:
This does break BC, but there's no way to reverse over a modulelist currently, so you would have to end up with something extremely convoluted to support it. I'm not sure that would be better than adding the `load_from_state_dict` logic. What we would end up would also be O(n^2) in # of blocks, which would get optimized away in script but would have performance implications in eager.